tests/extmod_hardware/machine_encoder.py

```diff
@@ -13,12 +13,22 @@
 import sys
 from machine import Pin
 
+PRINT = False
+PIN_INIT_VALUE = 1
```

Comment:
The test fails if this is set to any other value, yes? If so, I think it's OK to hard-code it at line 46